event_email_reminder/models/event.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2016 Tecnativa - Sergio Teruel
+# Copyright 2016 Tecnativa - Vicent Cubells
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+
+class EventEvent(models.Model):
+    _inherit = "event.event"
+
+    @api.model
+    def _send_event_template(self, events, template, partner_ids):
+        ctx = self.env.context.copy()
```

Comment:
I wonder if this is really necessary. Wouldn't a regular dict be enough as `with_context` merges the updated keys with the current context already?